app.js

```diff
@@ -34,15 +34,7 @@ app.use(session({
 }))
 
 app.use(function (req, res, next) {
-  var allowedOrigins = [
-    'http://localhost:8080',
-    'http://devpractical.com:5000',
-    'http://www.devpractical.com:5000',
-    'http://devpractical.com',
-    'http://www.devpractical.com',
-    'https://devpractical.com',
-    'https://www.devpractical.com'
-  ]
+  var allowedOrigins = process.env.PORD_DOMAIN
```

Comment:
I am not sure if you can embed a list of Strings as an environment variable. Please make sure this works locally and in prod.